build.sbt

```
@@ -33,11 +30,12 @@ resolvers ++= Seq(
   Resolver.bintrayRepo("webjars", "maven"),
   Resolver.typesafeRepo("releases"),
   Resolver.sbtPluginRepo("releases"),
-  Resolver.sonatypeRepo("releases"),
   Resolver.mavenLocal
 )
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-web" % "1.4.4")
+resolvers ++= Resolver.sonatypeOssRepos("snapshots")
+
+addSbtPlugin("com.github.sbt" % "sbt-js-engine" % "1.3.5-M1")
```

Please [upgrade](https://repo1.maven.org/maven2/com/github/sbt/sbt-js-engine_2.12_1.0/1.3.5/):
```suggestion
addSbtPlugin("com.github.sbt" % "sbt-js-engine" % "1.3.5")
```